security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -15,6 +15,21 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-3-0"]
+== OpenShift File Integrity Operator 1.3.0
+
+The following advisory is available for the OpenShift File Integrity Operator 1.3.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:3600[RHBA-2023:3600 OpenShift File Integrity Operator Bug Fix Update]
+
+* FIO now includes kubelet certificates as default files, excluding them from throwing warnings when they're managed by OpenShift.
```

Comment:
I would recommend "issuing" warnings instead of "throwing.

Also "... managed by {product-title}."

Reply:
Woops! Yes!